src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -45,4 +48,35 @@ public function helpAction(): Response
 
         return new Response();
     }
+
+    /**
+     * @Route("/comite/autocompletion",
+     *     name="app_citizen_project_committee_autocomplete",
+     *     condition="request.isXmlHttpRequest()"
+     * )
+     * @Method("GET")
+     * @Security("is_granted('IS_AUTHENTICATED_FULLY')")
+     */
+    public function committeeAutocompleteAction(Request $request)
+    {
+        if (!$request->query->get('term', false)) {
+            return new JsonResponse([], Response::HTTP_BAD_REQUEST);
+        }
+
+        $committees = $this
+            ->getDoctrine()
+            ->getRepository(Committee::class)
+            ->findByPartialName($request->query->get('term'));
+
+        $result = [];
```

Comment:
This can be removed then `return new JsonResponse($results ?? []);`

Reply:
✔️